### PR TITLE
My first change

### PR DESCRIPTION
**Pull request by LasagnaSenpai**
Funciona!

### CODE REVIEW

File: testGit.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world!
```
Comment (andreu-urruela) [inline testGit.txt:1]: I do not like this! ;)

Comment (andreu-urruela) [inline testGit.txt:1]: I would do something different....

**Review by andreu-urruela — COMMENTED** [review]
Not bad.....

File: testGit.txt (hunk 1 above)
Comment (LasagnaSenpai) [reply testGit.txt:1]: I dont care

**Pull request merged**